app/models/manageiq/providers/redhat/infra_manager.rb

```diff
@@ -43,187 +43,10 @@ def self.any_without_iso_datastores?
     without_iso_datastores.count > 0
   end
 
-  def supports_port?
-    true
-  end
-
-  def supported_auth_types
-    %w(default metrics)
-  end
-
   def supports_authentication?(authtype)
```

Comment:
Thank you! You are absolutely right. I will. Please tell me how to make sure that those PRs will be merged in the right order?